### ISSUE TITLE
Can't remove node, stuck at  	Waiting on user-node-remove_c-lsjl9

### ISSUE THREAD

**Issue by jdejean86** [rancher/rancher#39616]
Hello,

We tried to remove a node on rancher 2.5.16, but we still have "Waiting on user-node-remove_c-lsjl9".

Do you know why and what we can do to remove this node ?

Regards

**Comment by slickwarren** [comment]
We would need more information to fully understand the issue. But it is likely that some resource failed to delete, which you can search the local cluster resources for. 

2.5 is only receiving security related fixes at this time. It is possible that a later version of rancher has better logging or a fix for whatever issue you're seeing here. 

If you would like this specific issue debugged, please open a new ticket with the `bug` template filled out, pertaining to your cluster situation, reproduction steps, and any other helpful information.
